UKFast.API.Client.ECloud/Models/Request/CreateVirtualMachineRequest.cs

```diff
@@ -30,50 +31,53 @@ public class CreateVirtualMachineRequest
         [Newtonsoft.Json.JsonProperty("ram")]
         public int RAM { get; set; }
 
-        [Newtonsoft.Json.JsonProperty("hdd")]
-        public int HDD { get; set; }
+        [Newtonsoft.Json.JsonProperty("hdd", NullValueHandling = NullValueHandling.Ignore)]
+        public int? HDD { get; set; }
 
-        [Newtonsoft.Json.JsonProperty("hdd_disks")]
+        [Newtonsoft.Json.JsonProperty("hdd_disks", NullValueHandling = NullValueHandling.Ignore)]
         public IEnumerable<CreateVirtualMachineRequestDisk> HDDDisks { get; set; }
 
         [Newtonsoft.Json.JsonProperty("name")]
         public string Name { get; set; }
 
-        [Newtonsoft.Json.JsonProperty("computername")]
+        [Newtonsoft.Json.JsonProperty("computername", NullValueHandling = NullValueHandling.Ignore)]
         public string ComputerName { get; set; }
 
-        [Newtonsoft.Json.JsonProperty("tags")]
+        [Newtonsoft.Json.JsonProperty("tags", NullValueHandling = NullValueHandling.Ignore)]
         public IEnumerable<CreateTagRequest> Tags { get; set; }
 
-        [Newtonsoft.Json.JsonProperty("backup")]
-        public bool Backup { get; set; }
+        [Newtonsoft.Json.JsonProperty("backup", NullValueHandling = NullValueHandling.Ignore)]
+        public bool? Backup { get; set; }
```

Comment:
Default is `false` here, using `DefaultValueHandling = DefaultValueHandling.Ignore` rather than nullable would be better here